test/nieSyntaxSpecs.js

```diff
@@ -0,0 +1,42 @@
+describe('Testing nie syntax services', function() {
+	var $injector;
```

Comment:
I´m waiting for other reviewers opinión but in lc-validation-summary they dit it like this. I think it´s because this way you only create $injector variable once but if you do it in beforeEach you allocate memory each time you create the variable.